bundles/org.openhab.binding.warmup/src/main/java/org/openhab/binding/warmup/internal/api/MyWarmupApi.java

```diff
@@ -103,11 +103,39 @@ private void authenticate() throws MyWarmupApiException {
     public synchronized QueryResponseDTO getStatus() throws MyWarmupApiException {
         return callWarmupGraphQL("""
                 query QUERY { user { locations{ id name \
-                 rooms { id roomName runMode overrideDur targetTemp currentTemp \
-                 thermostat4ies{ deviceSN lastPoll }}}}}\
+                 rooms { id roomName energy runMode overrideDur targetTemp currentTemp fixedTemp \
+                 thermostat4ies{ deviceSN lastPoll airTemp floor1Temp floor2Temp }}}}}\
                 """);
     }
 
+    /**
+     * Call the API to set the room mode to program
+     *
+     * @param locationId Id of the location
+     * @param roomId Id of the room
+     * @param mode RoomMode defined in enum
+     * @throws MyWarmupApiException API callout error
+     */
+    public void setRoomMode(String locationId, String roomId, WarmupBindingConstants.RoomMode mode)
+            throws MyWarmupApiException {
+        if (mode == WarmupBindingConstants.RoomMode.SCHEDULE) {
+            callWarmupGraphQL(String.format("mutation{deviceProgram(lid:%s,rid:%s)}", locationId, roomId));
+        }
+    }
+
+    /**
+     * Call the API to set the room mode to fixed with a specific temperature
+     *
+     * @param locationId Id of the location
+     * @param roomId Id of the room
+     * @param temperature Temperature to set * 10
+     * @throws MyWarmupApiException API callout error
+     */
+    public void setFixed(String locationId, String roomId, int temperature) throws MyWarmupApiException {
+        callWarmupGraphQL(
+                String.format("mutation{deviceFixed(lid:%s,rid:%s,temperature:%d)}", locationId, roomId, temperature));
```

Comment:
Change to  `"mutation{deviceFixed(lid:%s,rid:%s,temperature:%d)}".formatted(locationId, roomId, temperature);`